src/context/user_context.js

```diff
@@ -18,91 +35,77 @@ export const UserProvider = ({ children }) => {
         .get(url, { withCredentials: true })
         .then((response) => {
           console.log(response);
-          setCurrentUser(response.data.user);
+          dispatch({ type: CURRENT_USER_FETCHED, payload: response.data.user });
```

Comment:
We are triggering the `CURRENT_USER_FETCHED` action/event, and passing along some information. Conventionally, we pass the information in a field called `payload` and the event in a field called \`type\` but it can be whatever you want.